cloud-sql.mk

```diff
@@ -0,0 +1,20 @@
+.PHONY: install-cloud-sql-proxy local-cloud-sql-config run-cloud-sql-proxy
+
+CLOUD_SQL_PORT ?= 3307
+
+install-cloud-sql-proxy:
+	test -f cloud_sql_proxy || \
+		curl -o cloud_sql_proxy https://dl.google.com/cloudsql/cloud_sql_proxy.darwin.amd64 && \
+		chmod +x cloud_sql_proxy
+
+local-cloud-sql-config:
+	mkdir -p batch-secrets
+	kubectl get secret batch-test-cloud-sql-config -n batch-pods -o "jsonpath={.data.batch-test-cloud-sql-config\.json}" | base64 --decode > batch-secrets/batch-test-cloud-sql-config.json
+	jq '.host = "127.0.0.1" | .port = $(CLOUD_SQL_PORT)' batch-secrets/batch-test-cloud-sql-config.json > batch-secrets/tmp.batch-test-cloud-sql-config.json
+	mv batch-secrets/tmp.batch-test-cloud-sql-config.json batch-secrets/batch-test-cloud-sql-config.json
```

Comment:
We can do it in one full pipe:
```
	kubectl get secret \
	    batch-test-cloud-sql-config -n batch-pods \
	    -o "jsonpath={.data.batch-test-cloud-sql-config\.json}" \
	  | base64 --decode \
	  | jq '.host = "127.0.0.1" | .port = $(CLOUD_SQL_PORT)' \
	    > batch-secrets/batch-test-cloud-sql-config.json
```

And I think I'd make this a non-PHONY target. If the secret changes, that will break folks, but I think that happening rarely is better than having to run `kubectl` every time we run the tests, right?